packages/context/test/acceptance/context-view.acceptance.ts

```diff
@@ -0,0 +1,182 @@
+import {expect} from '@loopback/testlab';
+import {
+  Context,
+  ContextView,
+  ContextEventListener,
+  Getter,
+  inject,
+  ContextEventType,
+  Binding,
+} from '../..';
+
+describe('ContextView - watches matching bindings', () => {
```

Comment:
I find the part `- watches matching bindings`, the same text is used in the test case name. Can you remove it from here please?

Reply:
Good catch.